src/usr/local/www/services_dyndns_edit.php

```diff
@@ -491,6 +492,9 @@ function setVisible(service) {
 				hideInput('ttl', false);
 				break;
 			case "namecheap":
+			case "cloudflare-v6":
+			case "cloudflare":
```

Comment:
Can we Remove this.   There already are lines controlling Cloudflare, and by adding these here the other lines don't get read in.

If this was intentional than can we remove the other cloudflare stanza?